Neos.NodeTypes/Migrations/TYPO3CR/Version20190304111200.yaml

```diff
@@ -0,0 +1,217 @@
+up:
```

Comment:
New Migrations should be added to `ContentRepository`- TYPO3CR is deprecated

Reply:
@daniellienert this was adjusted.